source/common/access_log/access_log_impl.h

```diff
@@ -173,13 +183,39 @@ class ResponseFlagFilter : public Filter {
   ResponseFlagFilter(const envoy::config::filter::accesslog::v2::ResponseFlagFilter& config);
 
   // AccessLog::Filter
-  bool evaluate(const StreamInfo::StreamInfo& info,
-                const Http::HeaderMap& request_headers) override;
+  bool evaluate(const StreamInfo::StreamInfo& info, const Http::HeaderMap& request_headers,
+                const Http::HeaderMap& response_headers,
+                const Http::HeaderMap& response_trailers) override;
 
 private:
   uint64_t configured_flags_{};
 };
 
+/**
+ * Filters requests that have a response with a gRPC status. Because the gRPC protocol does not
+ * guarantee a gRPC status code, if a gRPC status code is not available, then the filter will infer
+ * infer the gRPC status code if an HTTP status code if available.
```

Comment:
Also, nit: _if an HTTP status code_ -> _from an HTTP status code,_

Reply:
Changed in 52829ad.